### ISSUE TITLE
Over the wire 

### ISSUE THREAD

**Issue by gedw99** [soypat/natiu-mqtt#7]
Hey @soypat 

wow this looks pretty nice 

it looks like it can act as a control plane between different devices on a board ? 

I want to build a control plane gui so anyone can sniff the messages going on between the devices on a board. 

I would normally use nats which does support mqtt also btw but I saw hat someone has build a pure mqtt Server also that is pluggable ( Redis, mongo dB ).

looking for some advice from the expert dude :) 

I can also share my gui so you get a feel for what I am trying to do. The gui is meant to act as a debugger . It is multi window and streams all changes that it “hears” to the screen.

**Comment by soypat** [comment]
Yeah for sure you could use this as a control plane- you'd need a broker that all devices connect to though- natiu does not have a broker/server implementation. Several implementations out there like mosquitto. (saw a Go implementation recently that looked promising but can't seem to find the repo now!). The Broker would handle all this; persisting messages and whatnot.

Let me know whatever help you need setting up natiu dude!
